DIRECT_LINE_SPEECH.md

```diff
@@ -233,6 +233,8 @@ Direct Line Speech is not recommended to use on traditional websites where its p
 
 ## Requirements
 
+Direct Line Speech does not support Internet Explorer 11. It requires media capabilities that only modern browsers are capable.
```

Comment:
Direct Line Speech does not support Internet Explorer 11. It requires modern browser media capabilities that are not available in Internet Explorer.